### PR TITLE
Initial ObsMon infrastructure

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
This is PR begins building the basic infrastructure to make obs-monitor into a "model" as has been done with EVS.  These specific components have been added:

- j-job
- ex script
- driver/runObsMon.sh
- parm/OM_config

The driver script may be used to submit the j-job.  It has been tested only on hera.

Additional changes include:

- switch the use of "NET" to "MODEL", since, per EVS, NET should refer to this package.  
- change "RUN" to "COMPONENT", meaning ges | anl.

Completes #18

### CODE REVIEW

File: scripts/exobsmon_plot.sh
Comment (CoryMartin-NOAA) [inline scripts/exobsmon_plot.sh]: may want to make `python` into something like `APRUN_PY`? just so we can control it in case we need to provide the full path to the interpreter?

Comment (EdwardSafford-NOAA) [reply scripts/exobsmon_plot.sh]: Fair point.  I'll make that change directly.

Comment (EdwardSafford-NOAA) [reply scripts/exobsmon_plot.sh]: Done.

**Review by CoryMartin-NOAA — APPROVED** [review]
Looks good to me, want at least another set of eyes on it first before merging, thanks for this!

**Review by kevindougherty-noaa — APPROVED** [review]
Looks good to me. I don't see any glaring issues.

**Pull request merged**
